### DISCUSSION TITLE
[Idea] Add Sensors Info

### DISCUSSION THREAD

**Discussion by nchizhov** [glpi-project/glpi-agent#601] [Ideas]
### Your idea

How about add sensors to GLPI-Agent?

**Comment by g-bougard** [comment]
Hi @nchizhov 

you're talking about monitoring datas. GLPI-Agent is actually mainly an inventory agent. Few inventory datas like storage spaces could be considered as monitoring. But we are not sending them so often that they make sens from a monitoring point of view.

We don't have plan to make inventory evolving toward monitoring.
